_maps/RandomRuins/SpaceRuins/spacehotel.dmm

```diff
@@ -1802,12 +1796,11 @@
 /turf/open/floor/wood/tile,
 /area/ruin/space/has_grav/hotel)
 "qG" = (
-/obj/machinery/power/apc/highcap/five_k{
-	name = "Custodial APC";
-	pixel_y = -25
-	},
+/obj/machinery/power/apc/highcap/five_k/directional/south,
 /obj/structure/cable,
-/obj/structure/sink/directional/west,
+/obj/structure/sink/directional/west{
+	name = "Custodial APC"
+	},
```

Comment:
I don't think this var-edit is correct. You applied the name "Custodial APC" to the sink, not to the APC.

Reply:
<div class="tenor-gif-embed" data-postid="12353147" data-share-method="host" data-aspect-ratio="1.3369" data-width="100%"><a href="https://tenor.com/view/fizzer1k-5597-3689-move-along-nothing-to-seehere-gif-12353147">Fizzer1k 5597 GIF</a>from <a href="https://tenor.com/search/fizzer1k-gifs">Fizzer1k GIFs</a></div> <script type="text/javascript" async src="https://tenor.com/embed.js"></script>